File: practice/practice_a.cs
// https://atcoder.jp/contests/practice/submissions/7030932

using System;
using System.Linq;
class Program
{
    static void Main()
    {
        var a = int.Parse(Console.ReadLine());
        var input = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();
        var b = input[0];
        var c = input[1];
        var s = Console.ReadLine();
        Console.WriteLine($"{a + b + c} {s}");
    }
}

File: practice/practice_a.fs
// https://atcoder.jp/contests/practice/submissions/12059878

let a = stdin.ReadLine() |> int
let [| b; c |] = stdin.ReadLine().Split() |> Array.map int
let s = stdin.ReadLine()

a + b + c |> printf "%d "
s |> printfn "%s"
